lib/Registrator.js

```diff
@@ -164,11 +165,22 @@ ${this._contact};expires=${this._expires}${this._extraContactParams}`);
             const contacts = response.headers['Contact']
               .reduce((a, b) => a.concat(b.parsed), []);
 
-            // Get the Contact pointing to us and update the expires value accordingly.
-            const contact = contacts.find((element) => (
-              element.uri.user === this._ua.contact.uri.user
+            /* Get the Contact pointing to us and update the expires value accordingly.
+               If we have a reg-id and a sip.instance parameter in our Contact header then try to
+               find a matching Contact using that. */
```

Comment:
I know this is cosmetic but let's follow code convention:

```suggestion
            // Get the Contact pointing to us and update the expires value accordingly.
            // If we have a reg-id and a sip.instance parameter in our Contact header then try
            // to find a matching Contact using that.
```